### DISCUSSION TITLE
Clip Studio Paint cannot import SVGs exported from OpenToonz

### DISCUSSION THREAD

**Discussion by WolfInABowl** [opentoonz/opentoonz#6497] [Q&A]
I'm not sure what causes this issue as every other piece of software I've tried has no problem with OT's SVGs.
![image](https://user-images.githubusercontent.com/42905003/121789574-03f52b80-cbcf-11eb-9bdc-e26bc6548cd6.png)

Software that works without issue:
Blender, Krita, GIMP, Affinity Designer, Affinity Photo, Godot, FreeCAD

Re-saving the SVG from another software package, or using an online tool such as https://svgoptimizer.com, produces a file that CSP has no problem opening. The issue only seems to arise between CSP and OpenToonz.

**Comment by RodneyBaker** [comment]
As SVGs are text files it would be good to collect some example files that will and will not load in Clip Studio Pro.

Also, I'm not sure if you've queried the CSP community.
Perhaps they might be able to deduce the difference.

We do know that there have been some issues with the naming convention of the SVG files themselves that has been problematic but that doesn't seem to be case here.  Just to be sure though it'll be good to rule out name changes.
Attached is the code of a simple rectangle converted in OT to SVG (svg file with renamed extension .txt):

[rectangle.1.txt](https://github.com/opentoonz/opentoonz/files/6667030/rectangle.1.txt)
Note here that I changed the default from 'centerline' to 'outline'... perhaps that has something to do with the problem?

**Comment by RodneyBaker** [comment]
Apparently there are some specific restrictions with importing of SVG images into CSP.
REF:  https://www.clip-studio.com/site/gd_en/csp/userguide/csp_userguide/500_menu/500_menu_file_read_svg.htm#XREF_72855_PRO_EX
